nnfabrik/main.py

```diff
@@ -283,9 +351,10 @@ def add_entry(
         self,
         trainer_fn,
         trainer_config,
-        trainer_fabrikant=None,
         trainer_comment="",
+        trainer_fabrikant=None,
         skip_duplicates=False,
+        return_key_only=True,
```

Comment:
same thing here. doc is missing for the argument and the explanation for return should slightly change (to be similar to that of Dataset table)

Reply:
Updated